apps/frontend/src/app/5_pages/LendPage/components/AdjustModal/LendingForm.tsx

```diff
@@ -164,8 +164,13 @@ export const LendingForm: FC<DepositProps> = ({ state, onConfirm }) => {
       )}
       <SimpleTable className="mt-8">
         <SimpleTableRow
-          label={t(translations.lendingAdjust.newApy)}
-          value={<AmountRenderer value={newApy} suffix="% APY" />}
+          label={t(translations.lendingAdjust.newApr)}
+          value={
+            <AmountRenderer
+              value={newApy}
+              suffix={`% ${t(translations.lending.apr)}`}
```

Comment:
This looks really weird, pls verify with @john-light if we can display the value without the `LEND APR` suffix.
<img width="504" alt="image" src="https://github.com/DistributedCollective/sovryn-dapp/assets/14878713/cd661995-02ee-45b4-8d58-35fef2407b73">